src/util/hasValue.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Function that accepts data and determines whether that data has
+ * value or not
```

Comment:
this isn't correct, `undefined`, `null`, and `''` are all valid values in javascript- we should rather say in the description that it ensures that the value passed in is not one of {insert values we do not want to consider}

Reply:
done